### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get_display_info'

### ISSUE THREAD

**Issue by CitronSorcier** [arsenetar/dupeguru#272]
I got this error below:

```
Application Name: dupeGuru Picture Edition
Version: 2.10.1

Traceback (most recent call last):
  File "/usr/share/dupeguru_pe/qtlib/table.py", line 85, in data
    return self._getData(row, column, role)
  File "/usr/share/dupeguru_pe/qt/base/results_model.py", line 35, in _getData
    elif row.is_cell_delta(column.name):
  File "/usr/share/dupeguru_pe/core/gui/result_table.py", line 43, in is_cell_delta
    ref_info = self._group.ref.get_display_info(group=self._group, delta=False)
AttributeError: 'NoneType' object has no attribute 'get_display_info'
```

It happened after I've selected some files and sent them to Trash. This error occurred after I've tagged 'Delta values' on - it does not happen when Delta values are off.

Running Ubuntu 14.10 amd64.

**Comment by cbhushan** [comment]
I also got some very similar error with 'Delta values' on. Running on Linux Mint 17.1. 

```
Application Name: dupeGuru Picture Edition
Version: 2.10.1

Traceback (most recent call last):
  File "/usr/share/dupeguru_pe/qtlib/table.py", line 85, in data
    return self._getData(row, column, role)
  File "/usr/share/dupeguru_pe/qt/base/results_model.py", line 35, in _getData
    elif row.is_cell_delta(column.name):
  File "/usr/share/dupeguru_pe/core/gui/result_table.py", line 43, in is_cell_delta
    ref_info = self._group.ref.get_display_info(group=self._group, delta=False)
AttributeError: 'NoneType' object has no attribute 'get_display_info'
```

**Comment by scott-heterodyne** [comment]
Same soft crash, only on Windows 7 Pro

Also after having sent some to trash and set delta values
also, in case it matters, I notice that one of the filenames has odd characters (exclamation points and curly braces) - maybe causing problems with metacharacter expansion?

Application Name: dupeGuru Music Edition
Version: 6.8.1

Traceback (most recent call last):
  File "C:\src\dupeguru\qtlib\table.py", line 85, in data
  File "C:\src\dupeguru\qt\base\results_model.py", line 35, in _getData
  File "C:\src\dupeguru\core\gui\result_table.py", line 43, in is_cell_delta
AttributeError: 'NoneType' object has no attribute 'get_display_info'

**Comment by RoestVrijStaal** [comment]
I get the error in dupeGuru as well:

Application Name: dupeGuru
Version: 3.9.1

Traceback (most recent call last):
  File "C:\src\dupeguru\qtlib\table.py", line 85, in data
  File "C:\src\dupeguru\qt\base\results_model.py", line 35, in _getData
  File "C:\src\dupeguru\core\gui\result_table.py", line 43, in is_cell_delta
AttributeError: 'NoneType' object has no attribute 'get_display_info'

**Comment by SamDunham** [comment]
Got this today after deleting some files from the results.

Application Name: dupeGuru Picture Edition
Version: 2.10.1

Traceback (most recent call last):
  File "C:\src\dupeguru\qtlib\table.py", line 85, in data
  File "C:\src\dupeguru\qt\base\results_model.py", line 35, in _getData
  File "C:\src\dupeguru\core\gui\result_table.py", line 43, in is_cell_delta
AttributeError: 'NoneType' object has no attribute 'get_display_info'

**Comment by kindlydodo** [comment]
I have the same error here, at Windows 10 x64 in Version: 4.0.4 RC

Application Name: dupeGuru
Version: 4.0.4 RC

Traceback (most recent call last):
  File "qtlib\table.py", line 85, in data
  File "qt\results_model.py", line 39, in _getData
  File "core\gui\result_table.py", line 43, in is_cell_delta
AttributeError: 'NoneType' object has no attribute 'get_display_info'

**Comment by arsenetar** [comment]
This should be a duplicate error that has been fixed in #905, recommend using the latest version at this point.
